README_en.md

```
@@ -0,0 +1,165 @@
+| `--depth` | `0` | Maximum heading depth (`0` = unlimited) |
+| `--exclude` | - | Comma-separated heading texts to exclude (case-insensitive substring match) |
+| `--dry-run` | `false` | Print the result without writing to the file |
+| `--pretty` | `false` | In dry-run, render the whole formatted file in the terminal |
+
+Global flags: `--log-level` (`debug`, `info`, `warn`, `error`, `fatal`), `--log-format` (`text`, `json`) and `--log-no-colors`.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# Contributing
+
+Would you like to contribute? Great! There is a contribution guide to help you. Click [here](CONTRIBUTING.md) to read it.
```

markdownlint-cli2 (0.23.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)